pokemon-minisite/src/app/components/header/header.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="header">
-    <header></header>
+  <div *ngIf="isVerifiedSubject | async">
+    <button type="button" class="btn" (click)="logout()">Logout</button>
```

Comment:
try to remove the type button

Reply:
the style won't apply if I removed the type